src/lib/__tests__/server-utilities.test.js

```diff
@@ -72,3 +72,54 @@ describe('compressJPEG', () => {
     });
   });
 });
+
+describe('deleteFile', () => {
+  const source = './src/lib/__tests__/assets/copy-test.jpg';
+  const target = './tmp/copy-test.txt';
+
+  beforeAll(() => {
+    if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
+    return copyFile(source, target);
+  });
+
+  describe('successfully deletes file', () => {
+    expect.assertions(1);
+    if (fs.existsSync(target)) {
```

Comment:
Hmm, I'm not quite sure if I find this if statement a bit weird, but I guess the test will fail on `expect.assertions` anyway if the if statement doesn't work, as we definitely want the test to fail in that case.

What about doing `expect(fs.existsSync(target)).toBe(true)` instead?